### PR TITLE
[unit-tests-only] Add todo unit tests for PrivateLink view

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Added `todo` unit tests for `PrivateLink` view

## Related Issue
- part fo https://github.com/owncloud/web/issues/5237

## How Has This Been Tested?
- test environment: local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by pascalwengerter — COMMENTED** [review]
Looks good, please go ahead 🙌🏽

**Comment by saw-jan** [comment]
@pascalwengerter should I wait for this PR to be merged or add tests in this PR.
As we have discussed in an earlier meeting, we merge this PR and write actual tests in another PR

File: packages/web-app-files/tests/unit/views/PrivateLink.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+describe('PrivateLink view', () => {
+  it.todo('should have the background image set')
+  it.todo('should display the page title')
+  it.todo('should display the logo')
+  it.todo('should call "getPathForFileId" on mount')
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/views/PrivateLink.spec.js:5]:
```suggestion
  it.todo('should call "getPathForFileId" when the component is mounted')
```

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Please put the top 4 todos into their own "when the page has loaded successfully" `describe` block but otherwise good to go!

File: packages/web-app-files/tests/unit/views/PrivateLink.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+describe('PrivateLink view', () => {
+  describe('when the page has loaded successfully', () => {
+    it.todo('should have the background image set')
+    it.todo('should display the page title')
+    it.todo('should display the logo')
+    it.todo('should call "getPathForFileId" method')
+  })
+
+  describe('when "loading" is set as true', () => {
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/views/PrivateLink.spec.js:9]: I think 
```suggestion
  describe('when the page is still loading', () => {
```
would be a better description for the group

File: packages/web-app-files/tests/unit/views/PrivateLink.spec.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    it.todo('should display the loading text with the spinner')
+  })
+
+  describe('when there is error', () => {
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/views/PrivateLink.spec.js:13]:
```suggestion
  describe('when there was an error', () => {
```

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-app-files/tests/unit/views/PrivateLink.spec.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+describe('PrivateLink view', () => {
+  describe('when the page has loaded successfully', () => {
+    it.todo('should have the background image set')
+    it.todo('should display the page title')
+    it.todo('should display the logo')
+    it.todo('should call "getPathForFileId" method')
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/views/PrivateLink.spec.js:6]: I'm struggling with this description. I'd like to aim for a `test driven` development approach in the future. For that we'd have to write unit tests as black box tests. A test description about whether or not a certain function was called would be not allowed in such a case. Instead you'd have to describe behaviour of the component (`should resolve the provided file id to a path from the backend`).

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
Awesome 👍

**Review by pascalwengerter — APPROVED** [review]
LGTM, please go ahead!

**Pull request merged**